documentation/docs/developers/strategies/vwap.md

```diff
@@ -0,0 +1,53 @@
+# VWAP
+
+The VWAP strategy is a common algorithmic execution strategy which allows traders to account for slippage by splitting up orders over time. Specifically, the VWAP strategy helps traders minimize slippage when buying or selling large orders. The methods utilized in VWAP make the strategy more useful to traders and will help when creating future, more complex strategies.
+
+## Config
+
+VWAP utilizes user input for:
+
+* `is_vwap` : indicate if user wants to use VWAP or TWAP
+* `percent_slippage` : percent amount of order price that user is willing to set aside for slippage (0.1 signifies 0.1%)
+* `order_percent_of_volume` : percent of open order volume (at the specified price) that user wants to cap the order at (ex. user wants to cap the order volume at 0.1% of total available order volume)
+
+## Strategy
+
+The VWAP strategy attempts to limit the market impact of each individual user order. It accomplishes this by calculating the potential order sizes that can be placed in order to guarantee a user-specified upper bound on price slippage. In this strategy, the price slippage and corresponding order sizes are always calculated as if they are market orders, even if this is not the case. By doing do, the strategy achieves the goal of limiting the market impact of orders rather than limiting the actual price slippage of the orders.
+
+To achieve the goal of limiting the market impact of orders, the VWAP strategy fetches the order book and calculates the total open order volume up to percent_slippage. If no order is outstanding, an order is submitted which is capped at order_percent_of_volume * open order volume up to percent_slippage. The previous order is filled before the next is submitted and if an order is currently outstanding no action occurs.
```

Comment:
Code-quote `order_percent_of_volume * open order volume` and `percent_slippage`.